templates/effect-description-config.hbs

```diff
@@ -1,10 +1,11 @@
 <div class='form-group'>
+  <!--<label>game.i18n.localize("EffectDescription")</label>-->
```

Comment:
Foundry has a built-in Handlebar helper called `localize`. It can be used like this

```js
  /**
   * Translate a provided string key by using the loaded dictionary of localization strings.
   * @returns {string}
   *
   * @example Translate a provided localization string, optionally including formatting parameters
   * ```hbs
   * <label>{{localize "ACTOR.Create"}}</label> <!-- "Create Actor" -->
   * <label>{{localize "CHAT.InvalidCommand" command=foo}}</label> <!-- "foo is not a valid chat message command." -->
   * ```
   */
```

So in this case, it would be

```html
<label>{{localize 'EffectDescription'}}</label>
```